wrappers/kn/src/nativeMain/kotlin/zxingcpp/Barcode.kt

```diff
@@ -53,6 +53,23 @@ fun ZXing_Position.toKObject(): Position = Position(
 
 @OptIn(ExperimentalForeignApi::class)
 class Barcode(val cValue: CValuesRef<ZXing_Barcode>) {
+	companion object {
+		@ExperimentalWriterApi
+		fun fromText(text: String, opts: CreatorOptions): Barcode =
```

Comment:
I'd like to question the use of terms `fromText` and `fromBytes` in this context. Those are required if you have a language which either has no function overloading or you don't have a canonical type for those two options to overload on (that is true for C++, C and Rust, but e.g. not for C#). Here I would suggest to either simply have two (secondary) constructors that overload on `String` vs. `ByteArray` or if that is weird/impossible then maybe just a `from` function. If the constructor approach works, we also don't need the companion object, right?